src/components/Form/Form.Styled.ts

```diff
@@ -33,20 +34,39 @@ export const StyledDiv = styled.div`
 	align-items: flex-start;
 	width: 315px;
 	margin-bottom: 4em;
-	border: 2px solid #cecece;
-	border-radius: 10px;
-	overflow: hidden;
+`;
 
-	&:focus {
-		border: 3px solid white;
-	}
+export const StyledLabel = styled.label`
+	display: flex;
+	padding: 1em 1em;
+	font-size: ${(props) => props.theme.fontSize.normal};
+	text-transform: uppercase;
+	color: ${(props) => props.theme.colors.pureWhite};
+	overflow: hidden;
 `;
 
 export const StyledInput = styled.input`
+	display: flex;
 	width: 100%;
 	font-size: ${(props) => props.theme.fontSize.normal};
 	padding: 0.5em 1em;
 	background-color: transparent;
+	color: ${(props) => props.theme.colors.pureWhite};
 	border-radius: 5px;
+	border: 2px solid transparent;
 	overflow: hidden;
+
+	&:focus {
+		border: 2px solid ${(props) => props.theme.colors.pureWhite};
+	}
+`;
+
+export const StyledError = styled.p<{ text?: string; error?: boolean }>`
+	position: absolute;
+	display: ${(error) => (error ? "flex" : "none")};
+	left: 20px;
+	top: 90px;
```

Comment:
Oj nie panie kolego, na pewno errora nie zrobimy absolutnie! :P 
To musi być zwykły element który znajduje się standardowo w strukturze. Wyobraź sobie że masz dłuższy message który się rozleje na dwie linijki, prawdopodobnie najedzie on na dolny input. Moze sie to zdarzyć na mobilce, lub po prostu na dluzszym message.

Jeżeli chcesz uniknąć "skakania" formularza gdy wyskakuje error (bo one sie pojawią, zajmą miejsce i całośc zjedzie) to możesz mu ustawić minimalną wysokość równą jednej linijce i display: none robić dla contentu, ale wrapper niech trzyma wysokość na zapas.